### PR TITLE
Introduction updated in documentation

### PR DESCRIPTION
**Pull request by MatteoRobbiati**
In this PR I propose two modifications to the doc:

1. created a "OS and Python support" paragraph into the `Installation instructions`;
2. created a new subsection into the `Getting started` section in which we can explain the relationship between ``qibocal`` and ``qibolab``, asking explicitly to the user to install the `tiiq` flag together with ``qibolab``.

### CODE REVIEW

File: doc/source/getting-started/index.rst [hunk 1]
```diff
@@ -2,10 +2,13 @@ Getting started
 ===============
 
 In this section we present the basic aspects of the Qibocal design and provide installation instructions.
+Please visit the following sections to understand how ``qibocal`` works and how you can prepare and calibrate
+hardware so that you can improve the execution of your algorithms.
```
Comment (andrea-pasquale) [inline doc/source/getting-started/index.rst:6]:
```suggestion
your quantum hardware.
```

File: doc/source/getting-started/index.rst [hunk 2]
```diff
@@ -2,10 +2,13 @@ Getting started
 ===============
 
 In this section we present the basic aspects of the Qibocal design and provide installation instructions.
+Please visit the following sections to understand how ``qibocal`` works and how you can prepare and calibrate
```
Comment (andrea-pasquale) [inline doc/source/getting-started/index.rst:5]:
```suggestion
Please visit the following sections to understand how ``qibocal`` works and how you can characterize and calibrate
```

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @MatteoRobbiati.
Please find below some comments.

File: doc/source/getting-started/qibolab4qibocal.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Qibocal needs Qibolab!
+==============================
+
+The ``qibocal`` package works closely with ``qibolab``.
```
[the remark this answers is not in the document]
Comment (andrea-pasquale) [reply doc/source/getting-started/qibolab4qibocal.rst:4]: @MatteoRobbiati if we update this part we should be able to merge this PR? I can also do it if you want.

**Comment by andrea-pasquale** [comment]
@Edoardo-Pedicillo could you please review this PR?

**Comment by Edoardo-Pedicillo** [comment]
1. it is ok for me 
2. I suggest moving the "Qibocal needs Qibolab!" section to "installation and instructions", or at least putting a direct link in "Installation and instructions", to avoid misunderstandings among TII people who want to install Qibocal.

**Review by Edoardo-Pedicillo — APPROVED** [review]

File: doc/source/getting-started/qibolab4qibocal.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (MatteoRobbiati) [reply doc/source/getting-started/qibolab4qibocal.rst]: @andrea-pasquale I've removed the 4th section from the `getting-started`. I followed @Edoardo-Pedicillo idea, including it into the installation instructions and i changed the message in order to follow your suggestion.

What do you think?

[the remark this answers is not in the document]
Comment (andrea-pasquale) [reply doc/source/getting-started/qibolab4qibocal.rst]: Thanks @MatteoRobbiati looks good to me now.

**Pull request merged**